Gemfile

```diff
@@ -119,3 +119,5 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem "phonelib"
+
+gem 'cancancan'
```

Comment:
Would you want to add a comment to the `Gemfile` on what this particular dependency accomplishes? Maybe a link to how to interact with it would also help others, or a reference to a project file that shows correct usage?